includes/AddOns/ConcreteDesign/ConcreteDesignSupport.js

```diff
@@ -0,0 +1,1114 @@
+include("../../Tools/jshlf_common_functions.js");
```

Comment:
Please consider renamig of the file - misleading name - it contains auxiliary functions but looks like something else.

Reply:
File removed, function EnumValueFromJSHLFTypeName moved to high_level_functions_support.js.